pipert2/utils/shared_memory/__init__.py

```diff
@@ -0,0 +1,6 @@
+import sys
+
+if sys.version_info.minor <= 7:
+    from pipert2.utils.shared_memory.posix_ipc.shared_memory_manager import SharedMemoryManager
+else:
+    from pipert2.utils.shared_memory.multiprocessing.shared_memory_manager import SharedMemoryManager
```

Comment:
Although they have the same methods names, maybe they should implement SharedMemoryInterface to force them implement the same format.